index.html

```diff
@@ -224,5 +224,71 @@ <h3>Skills</h3>
         </ul>
       </div>
     </div>
+
+    <section class="contact-form">
+      <div class="contact-header">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+      </div>
+      <form
```

Comment:
- [ ] Please, add a `action` tag with the [formspree link](https://formspree.io/html). Remember that you need to use it on your form according to the [Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Collect%20data%20by%20using%20the%20Formspree%20service.) 🐶